### DISCUSSION TITLE
Compability with Next.JS?

### DISCUSSION THREAD

**Discussion by CrispinMartens** [AntoineW/luge#9] [Q&A] [answered]
Hello there 👋

I Stumbled upon Luge, and would love to give it a shot right away since I would have continuous use for it, working on a broad universe of landing pages and microsites where the ability to throw in nice animations fast would be a huge asset..

.. but is Luge compatible with Next.JS? I gave an integration two attempts, one on a local Next.JS project and one on codesandbox.io but failed to get both to work.

**Comment by AntoineW** [comment, marked as answer]
Hey,
Yes luge latest version should work with Next.js. As said in #10 issue I created a sandbox to show how to make it works here https://codesandbox.io/s/magical-ramanujan-3njk4

Let me know if it's not clear enough!
